logic.py

```
@@ -672,7 +709,66 @@ def _is_candidate_valid(col):
+        if mcc_enable and aor_results:
+            mv_p_vals = []
+            mv_keys = []
+            for k, v in aor_results.items():
+                p = v.get('p_value')
+                if pd.notna(p):
+                     mv_p_vals.append(p)
+                     mv_keys.append(k)
+            
+            if mv_p_vals:
+                adj_p = apply_mcc(mv_p_vals, method=mcc_method)
+                for k, p_adj in zip(mv_keys, adj_p):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)